src/modules/navigation/AppDrawerNavigator.js

```diff
@@ -1,43 +1,109 @@
 import React from 'react';
-import { createDrawerNavigator } from 'react-navigation-drawer';
-import { MaterialIcons } from '@expo/vector-icons';
+import DrawerNavigator from 'react-navigation-drawer/src/navigators/createDrawerNavigator';
+import CustomDrawerContentComponent from '../../components/navigation/CustomDrawerContentComponent';
+import WhatToTakeView from '../../views/drawerNavigatorScreens/WhatToTakeView';
+import PartnersView from '../../views/drawerNavigatorScreens/PartnersView';
+import StartPackageView from '../../views/drawerNavigatorScreens/StartPackageView';
+import ContactView from '../../views/drawerNavigatorScreens/ContactView';
+import AboutAppView from '../../views/drawerNavigatorScreens/AboutAppView';
+import LogoutView from '../../views/drawerNavigatorScreens/LogoutView';
 import DashboardScreen from '../../views/dashboardScreen/DashboardScreen';
-import AppBottomNavigator from './AppBottomNavigator';
+import { Image } from 'react-native';
 import { ksStyle } from '../../styles/basic/ksBasic';
+import { DrawerItem } from '@react-navigation/drawer';
 
-const IconComponent = MaterialIcons;
-
-export default createDrawerNavigator(
+export default new DrawerNavigator(
   {
-    Bottom: AppBottomNavigator,
-    Partners: {
+    Dashboard: {
       screen: DashboardScreen,
       navigationOptions: {
+        drawerLabel: () => null,
+      },
+    },
+    WhatToTake: {
+      screen: WhatToTakeView,
+      navigationOptions: {
+        drawerIcon: (
+          <Image
+            source={require('../../../assets/icons/backpack.png')}
+            style={ksStyle.drawerIcon}
+          />
+        ),
+        drawerLabel: 'Co zabrać?',
+      },
```

Comment:
Imo we can define those objects elsewhre, so the Navigator can stay nice and compact

Reply:
You're right. I will put config in different file.